src/tests/riscv64_zkn.s

```diff
@@ -0,0 +1,10 @@
+/* RISC-V - test if the Zkn extension is present */
```

Comment:
This file is not referenced in CMakeLists.txt, and I don't see an `rx_aesenc_vec_i128` implementation. Is hardware AES not implemented yet?

Reply:
Yes, this PR does not include hardware AES. I removed the file for now because it's unlikely to be needed in the near future.